js/ddmlib/ddmclient.js

```diff
@@ -340,6 +340,26 @@ class OfflineServiceController {
     }
 }
 
+class TimeLapseBugReportServiceController {
+    constructor(appInfo) {
+        this.bugReportLine = appInfo.data
+    }
+
+    loadViewList() {
+        const base64String = this.bugReportLine.replace(VIEW_CAPTURE_REGEX, "")
+
+        return protobuf.load("../protos/view_capture.proto").then(function (root) {
```

Comment:
This part can probably be moved to worker.js so that the UI thread is not blocked

Reply:
I'll work on moving this code into worker.js in a follow-up CL.